Babylon/commands/macro/apply.py

```diff
@@ -20,11 +20,18 @@
 @command()
 @injectcontext()
 @argument("deploy_dir", type=Path(dir_okay=True, exists=True))
-def apply(deploy_dir: pathlib.Path):
+@option('--var-file',
+        'variables_files',
+        type=Path(file_okay=True, exists=True),
+        default=["./variables.yaml"],
+        multiple=True,
+        help='Specify the of your variable file. By default, it takes the variables.yaml file.')
```

Comment:
ajouté ici dans help message `Specify the path of your variable file. By default, it takes the variables.yaml file`